src/app/preferences/preferences.module.ts

```diff
@@ -25,4 +28,4 @@ namespace Core {
     .service('preferencesRegistry', PreferencesRegistry)
     .name;
 
-}
+}
```

Comment:
Can you install EditorConfig extension for VS Code?
https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
It will automatically add line break at the end of file.

Reply:
Not resolved yet.